core/src/main/java/org/elasticsearch/action/admin/cluster/settings/SettingsUpdater.java

```diff
@@ -49,15 +49,12 @@ synchronized Settings getPersistentUpdate() {
     }
 
     synchronized ClusterState updateSettings(final ClusterState currentState, Settings transientToApply, Settings persistentToApply) {
-        boolean changed = false;
         Settings.Builder transientSettings = Settings.builder();
         transientSettings.put(currentState.metaData().transientSettings());
-        changed |= clusterSettings.updateDynamicSettings(transientToApply, transientSettings, transientUpdates, "transient");
-
         Settings.Builder persistentSettings = Settings.builder();
         persistentSettings.put(currentState.metaData().persistentSettings());
-        changed |= clusterSettings.updateDynamicSettings(persistentToApply, persistentSettings, persistentUpdates, "persistent");
-
+        boolean changed = clusterSettings.updateDynamicSettings(transientToApply, transientSettings, transientUpdates, "transient") |
```

Comment:
Why do the OR operation on 2 methods in 3 lines when we can simply do the same in one line.
In the earlier way we had to do the OR operation twice.